app/upgrades/v16/upgrades.go

```diff
@@ -128,6 +128,12 @@ func CreateUpgradeHandler(
 
 		updateTokenFactoryParams(ctx, keepers.TokenFactoryKeeper)
 
+		// Transfers out all the dev fees in kvstore to dev account during upgrade
+		// Expceted to error during e2e test due to dev account not being set
+		if err := keepers.ProtoRevKeeper.SendDeveloperFeesToDeveloperAccount(ctx); err != nil {
+			ctx.Logger().Error("error sending developer fees to developer account during upgrade, expected in testing, not in prod", "error", err)
+		}
```

Comment:
I would prefer it if this is handled properly. I know the process for updating e2e is cumbersome. However, handling this properly will save us time during the upgrade if any issues occur.

The process for setting this account in e2e is the following:

1. update e2e genesis for the protorev module in a separate PR: https://github.com/osmosis-labs/osmosis/blob/1e56cc620a17e0b5debb8228896aa44b6421a723/tests/e2e/initialization/config.go#L234-L237
2. backport that PR to `v15.x`
3. Grab the chain init docker image with v15 tag from here: https://hub.docker.com/r/osmolabs/osmosis-e2e-init-chain/tags
  * should be auto-uploaded
4. Copy this tag: https://github.com/osmosis-labs/osmosis/blob/1e56cc620a17e0b5debb8228896aa44b6421a723/tests/e2e/containers/config.go#L27

Reply:
hi @NotJeremyLiu is this PR r4r now. I think just needs rebasing right?

Reply:
@stackman27 I just updated the tag and removed the failing test, letting it run through workflow to see if all is good